include/arch/x86/page.h

```diff
@@ -117,6 +117,7 @@
 #define PADDR_MASK  (~(PADDR_SIZE - 1))
 
 #define VIRT_KERNEL_BASE _U64(0xffffffff80000000)
+#define VIRT_KERNEL_HIGH _U64(0xffff900000000000)
```

Comment:
this is actually VIRT_KERNEL_LOW
VIRT_KERNEL_BASE is at higher address numerically.

I believe that's the same convention used in most kernels.

In our case I don't think we need high.
It seems like we really need new address space in negative addr space.

Can we rename it to something else to avoid that confusion?

Reply:
> this is actually VIRT_KERNEL_LOW
VIRT_KERNEL_BASE is at higher address numerically.

For the area yes. But for the allocations with higher addresses or sizes the name describes it better IMO. Would not insist on the name though. 

> I believe that's the same convention used in most kernels.

> In our case I don't think we need high.
It seems like we really need new address space in negative addr space.

Yes, we need a range where memory allocations of big sizes or higher numerical address values could be mapped. Hence the name. 

> Can we rename it to something else to avoid that confusion?

Happy to rename. Suggestions welcome.

Reply:
Spoke with Pawel on this too earlier yesterday. Agree the name could be better and agree this name is anyway fine

Reply:
I renamed to VIRT_KERNEL_MAP. Hope it is better.

Reply:
Sounds good to me.